pkg/chain/bitcoin/handle.go

```diff
@@ -3,6 +3,29 @@ package bitcoin
 // Handle serves as an interface abstraction around bitcoin network queries
 type Handle interface {
 	Broadcast(transaction string) error
-	VbyteFee() (int32, error)
+	VbyteFeeFor25Blocks() (int32, error)
 	IsAddressUnused(btcAddress string) (bool, error)
 }
+
+type OfflineHandle struct{}
```

Comment:
Looking at this more closely, I'm wondering if it's the right strategy. We can substitute in the `OfflineHandle` if we see the URL is missing, but the caller will still need to handle situations where we have an `ElectrsConnection` _and_ it fails to return correctly from the API. I think these are the same behaviors, in which case having them in one place (the caller) rather than two (the caller + `OfflineHandle`) seems better.

Let me know if I've misunderstood how the pieces are fitting together though.

Reply:
Summarizing from an offline conversation:

* We have a couple of choices here. Given that currently, we have a single caller of off of these methods (client.go), and that caller is in charge of any retry/fallback logic, we need to figure out where to draw the boundaries in responsibilities.

As written, the caller (client.go) is responsible for fallback/retry logic, and the bitcoin handle has a low-level single-try mechanism. Instead, we could put the fallback/retry logic in the handle and privatize the single-try stuff.

Both mildly violate YAGNI (you aren't going to need it):

* If we abstract the retry logic into the handler, and move the api calls to a lower-level function, then we're making an assumption that some other caller in the future will need that same retry logic (which we aren't sure of yet).
* If we leave the retry logic in the caller, and preserve the low-level single-try logic, then we're making an assumption that we're going to need a non-retry version of that function call in the future (which we aren't sure of yet).

In the interest of getting code out of the already-very-bloated client.go, I'm going to see how this looks with the retry/fallback logic in the handle